app/services/weets.js

```diff
@@ -0,0 +1,7 @@
+const api = require('../../config/axios');
+
+const fetchWeet = () => api.get('/random');
```

Comment:
Here you should do some error caching and handling.
Also, normally you would have some data mapper here, because in most cases you don't want to directly return what the external api responds to you api

Reply:
i had made changes for the error catching and handling but im no sure how to implement the mapper since it only returns the information needed for the weet, if you could explain a little more I would appreciate it.